subsys/mgmt/fmfu/src/fmfu_mgmt.c

```diff
@@ -219,14 +218,8 @@ static struct mgmt_group fmfu_mgmt_group = {
 
 int fmfu_mgmt_init(void)
 {
-	struct nrf_modem_full_dfu_digest digest;
+	struct nrf_modem_bootloader_digest digest;
 
-	int err = nrf_modem_full_dfu_init(&digest);
-
-	if (err != 0) {
-		LOG_ERR("nrf_fmfu_init failed, err: %d\n.", err);
-		return err;
-	}
 	mgmt_register_group(&fmfu_mgmt_group);
 	return err;
```

Comment:
CI: `err` is undeclared.
```suggestion
	return 0;
```

Reply:
Thanks, fixed